components/subscriptions-report/SubscriptionsReport.module.css

```
@@ -0,0 +1,14 @@
+@keyframes upcomingRowIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "upcomingRowIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)